lib/fastlane/actions/slather.rb

```diff
@@ -32,7 +32,7 @@ def self.run(params)
         command += " --show" if params[:show]
         command += " --source-directory #{params[:source_directory]}" if params[:source_directory]
         command += " --output-directory #{params[:output_directory]}" if params[:output_directory]
-        command += " --ignore #{params[:ignore]}" if params[:ignore]
+        command += " #{params[:ignore].split('|').map { |path| "--ignore \"#{path}\"" }.join(' ')}" if params[:ignore]
```

Comment:
this should be able to take in a legit array instead of having to split a string.